assets/src/application/services/toaster/useSystemNotificationsManager.tsx

```diff
@@ -0,0 +1,66 @@
+import { useCallback } from 'react';
+
+// import useLoadingToast from './useLoadingToast';
+import useToaster from './useToaster';
+import { AddToast, DissmissToast, SystemNotificationsManager, TOAST_STATUS } from './types';
+
+const useSystemNotificationsManager: SystemNotificationsManager = (options?) => {
```

Comment:
this allows the base toaster to be used for other kinds of notifications.

So System Notifications will appear in the bottom right corner of the screen primarily for providing feedback for CRUD operations,
but then we could add another type of notifications that could appear somewhere else, like at the top of the screen and be used for some other purpose